.golangci.yml

```diff
@@ -46,6 +46,11 @@ linters-settings:
           - "!**/internal/handlers/hana/hanadb/*.go"
         deny:
           - pkg: github.com/FerretDB/FerretDB/internal/handlers/sjson
+      backends:
+        files:
+          - "**/internal/backends/**/*.go"
```

Comment:
Is ` **/internal/backends/*.go` also included?

Reply:
`**` expands into anything, including an empty string and sub/dir/ectory